pkg/cmd/experimental/syncgroups/grouppruner_test.go

```diff
@@ -0,0 +1,106 @@
+package syncgroups
+
+import (
+	"errors"
+	"io/ioutil"
+	"testing"
+
+	ktestclient "k8s.io/kubernetes/pkg/client/unversioned/testclient"
+	"k8s.io/kubernetes/pkg/runtime"
+	"k8s.io/kubernetes/pkg/util/sets"
+
+	"github.com/openshift/origin/pkg/client/testclient"
+	"github.com/openshift/origin/pkg/cmd/experimental/syncgroups/interfaces"
+)
+
+func TestGoodPrune(t *testing.T) {
```

Comment:
Tests like this are a pain, because I have to know the state that various test helpers work with in order to determine whether this test is doing the right thing.  Things that declare their starting and ending data are much easier to follow.